internal/storage/postgreSQL/treeSQL/tree.go

```diff
@@ -0,0 +1,85 @@
+package treeSQL
```

Comment:
I would name it only `tree`
```suggestion
package tree
```